scripts/build-profiles/stencil-component/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@okta/build-profiles.stencil-component",
+  "description": "Tools to build, run, and test a stencil component",
+  "private": true,
+  "version": "0.0.0",
+  "bin": {
+    "stencil-component": "./bin/stencil-component.js"
+  },
+  "dependencies": {
+    "@okta/scripts.logger": "0.1.0-beta.4322.g65a2415",
```

Comment:
This is another internal dependency we will not have access to - so let's remove it.